src/oneTrust/getListOfAssessments.ts

```diff
@@ -0,0 +1,49 @@
+import { Got } from 'got';
+import { logger } from '../logger';
+import {
+  OneTrustAssessment,
+  OneTrustGetListOfAssessmentsResponse,
+} from './types';
+
+/**
+ * Fetch a list of all assessments from the OneTrust client.
+ *
+ * @param param - the information about the OneTrust client
+ * @returns a list of OneTrustAssessment
+ */
+export const getListOfAssessments = async ({
+  oneTrust,
+}: {
+  /** The OneTrust client instance */
+  oneTrust: Got;
+}): Promise<OneTrustAssessment[]> => {
+  let currentPage = 0;
+  let totalPages = 1;
+  let totalElements = 0;
+
+  const allAssessments: OneTrustAssessment[] = [];
+
+  logger.info('Getting list of all assessments from OneTrust...');
+  while (currentPage < totalPages) {
+    // eslint-disable-next-line no-await-in-loop
+    const { body } = await oneTrust.get(
```

Comment:
nit sjould use io-ts codec validation here

Reply:
will do in a follow up PR as well!